app/helpers/office_schedule_helper.rb

```diff
@@ -0,0 +1,7 @@
+module OfficeScheduleHelper
+  def get_weekly_office_assignments(week, office)
+    assignments = Assignment.where(week: week, office: office)
+
+    return assignments
```

Comment:
Could do a implicit return of `Assignment.where(week: week, office: office)` because  the value of the last evaluated expression will be returned. On the other hand this is nice and explicit